pom.xml

```diff
@@ -16,7 +16,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>6.3.0</version>
+        <version>6.5.0-SNAPSHOT</version>
```

Comment:
I know that I said the opposite a few days ago so please @davidfestal apologies...but thinking more about that what about setting 6.4.1 here and ask Roman to release a 6.4.1 that would include https://github.com/eclipse/che/commit/c66d34972816f2d8684dcca34e4b1592e1f2e95e?

Reply:
No problem. I'll test it on the 6.4.0 branch, and will ask Roman to create the release when it works correctly.

Reply:
Thank you @davidfestal

Reply:
done in commit https://github.com/redhat-developer/rh-che/pull/646/commits/3726f5b1ada13e5ad3e5c04e2205973982177879